app/util/data_preparation/jsd_prepare_data.py

```diff
@@ -42,6 +49,7 @@ def wrapper(*args, **kwargs):
             result = func(*args, **kwargs)
             end = timer()
             print(f"{message} finished in {timedelta(seconds=end-start)} seconds")
+            print('------------------------------------------------------')
```

Comment:
I have seen that you are removing this print below, maybe you forgot about this print?

Reply:
Agree, lets review it after finishing jsd developing as well as PEP8 fixes